### ISSUE TITLE
[Bug]: auction house listing asset error SaleRequiresExactlyOneSigner

### ISSUE THREAD

**Issue by boredjoker** [metaplex-foundation/metaplex-program-library#1049]
### Which package is this bug report for?

auction-house

### Which Type of Package is this bug report for?

Rust Contract

### Issue description

when auction house requires_sign_off is true, how can we listing a aseet by user, signing with authority,and send tranction by user(as fee payer),why can't we let user and authority as both signer like last version?

1. last version success [tranction](https://solscan.io/tx/D4dBerAbcCefmwZUHD1zG54qFa9B4U8fiCSJBJvtvajx6WjBDxcQUi2q5jfUAeyhjs1yiErCgSC7odSrJ1JhMPg?cluster=devnet)

2. new version error [tranction](https://solscan.io/tx/53GowrhYMnZaXeFo6v5VhmY3tmwLGD83Uz8E4Uny41HNgR4o6yJcsF43mZ1xsReMU8VH2iYNq4tByF9zDjaqfSVb?cluster=devnet)
Error Code: SaleRequiresExactlyOneSigner. Error Number: 6044. Error Message: This sale requires exactly one signer: either the seller or the authority..

3. new version error without authority sign  [tranction](https://solscan.io/tx/4YyPPEznUYLaTuMrhQqZnTd1nWVsjtRDRchDQsg3CJhEcVAhh1dNYuy9BZbR8LzeR9DozE156MoHM7ogY8WGD33Q?cluster=devnet)
Error Code: CannotTakeThisActionWithoutAuctionHouseSignOff. Error Number: 6011. Error Message: Cannot take this action without auction house signing too.

### Priority this issue should have

High (immediate attention needed)

**Comment by carrie-hk** [comment]
Hey we're getting the same error! Any update on this?

**Comment by jdnichollsc** [comment]
Hello @samuelvanderwaal, do you need more details about this error? Thanks for your help, please let us know how can we help! <3

**Comment by samuelvanderwaal** [comment]
@boredjoker @carrie-hk @jdnichollsc hey everyone, I'm opening a PR to remove the double signer restriction as there does not appear to be a good reason to keep that in place. Keep in mind that the fee payer still defaults to the auction house fee account with two signers and it's out of scope to refactor this as it affects more than just this handler. However, to work around this you can include a simple system transfer instruction from the user to the auction house fee account if you want the user to pay for the creation of the trade state account.

**Comment by carrie-hk** [comment]
Also @samuelvanderwaal there's a version mismatch on the deployed program see [here](https://explorer.solana.com/address/hausS13jsjafwWwGqZTUQRmWyvyxn9EQpqMwV1PBBmk/anchor-program?cluster=devnet&customUrl=http%3A%2F%2Flocalhost%3A8891). It should be v1.4.0, but says v1.3.4

**Comment by carrie-hk** [comment]
This problem isn't in v1.3.4, but when we're calling on devnet we are facing this error! Not too sure what's up there -- maybe an IDL mismatch?

**Comment by samuelvanderwaal** [comment]
> This problem isn't in v1.3.4, but when we're calling on devnet we are facing this error! Not too sure what's up there -- maybe an IDL mismatch?

Yeah probably just an IDL mismatch. I will bump versions and redeploy to both devnet and mainnet today.
